src/WebJobs.Script/EnvironmentSettingNames.cs

```diff
@@ -23,5 +21,6 @@ public static class EnvironmentSettingNames
         public const string AzureWebJobsExtensionsPath = "AzureWebJobs_ExtensionsPath";
         public const string AzureWebsiteAppCountersName = "WEBSITE_COUNTERS_APP";
         public const string AzureWebJobsSecretStorageType = "AzureWebJobsSecretStorageType";
+        public const string AppInsightsInstrumentationKey = "APPINSIGHTS_INSTRUMENTATIONKEY";
```

Comment:
This is already the key you've been using, right? Wondering if this should use our standard prefix to scope to functions and avoid conflicting with something else that decides to use `APPINSIGHTS_INSTRUMENTATIONKEY` in the future.